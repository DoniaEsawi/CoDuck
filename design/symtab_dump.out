------------------------ ------------ ------------------
Name                     Type         Line Number       
------------------------ ------------ ------------------
a                        UNDEF        5      
i                        UNDEF        1      
hello                    UNDEF        3      
