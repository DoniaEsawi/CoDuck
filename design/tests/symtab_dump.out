------------------------ ------------ ------------------
Name                     Type         Line Number       
------------------------ ------------ ------------------
val                     REAL             0  2      3      4      
i                       INT              0  1      
res                     REAL             0  2      4      
